docs/one-click-integrations/github-integration.md

```diff
@@ -139,6 +139,58 @@ The table below lists the data that the GitHub integration collects from your Gi
 
 <sup><span id="deployment-teams">2</span></sup>: Adding a new team or changing the composition of a team on GitHub only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted teams on GitHub will continue to show in Pulse.
 
+## Which permissions does Pulse need from GitHub? {: id="gh-permissions"}
+
+Pulse requests only the necessary permissions from GitHub to collect your organization changes and deployment data and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
+
+<table>
+  <colgroup>
+    <col width="20%"/>
+    <col width="20%"/>
+    <col width="60%"/>
+  </colgroup>
+  <thead>
+    <tr>
+      <th>Scope</th>
+      <th>Permissions</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td colspan="3"><strong>Repository permissions:</strong></td>
+    </tr>
+    <tr>
+      <td>Pull requests</td>
+      <td>Read</td>
+      <td>Pulse retrieves pull request information to calculate several of the metrics presented on the dashboards. <a href="#collected-data"> See the details here.</a></td>
+    </tr>
+    <tr>
+      <td>Contents</td>
+      <td>Read</td>
+      <td>Pulse retrieves contents of files and directories in a repository to read tags and support semantic versioning.</td>
```

Comment:
while we ask for permission to "retrieves contents of files and directories in a repository" we don't use it, and we can make it more clear to users

we only used this permission to read git tags which enable the support of the semantic version tags option in the Integration Settings. unfortunately, the permission to read git tags is the same to get access to all files of a repository

Reply:
Updated. See how it looks.